otb-luci/files/usr/lib/lua/luci/view/otb_overview.htm

```diff
@@ -72,6 +94,219 @@ <h2><%:Overview%></h2>
         return ret + seconds + ' seconds';
     }
 
+    /* start of otb-get-data section */
+    XHR.poll(15, '<%=url("admin/overthebox/data")%>', null, function(x,p) {
+        _otb_data = p;
+    });
+    function otb_get_data(...args) {
+        while(_otb_data.time == 0);
+        var data = _otb_data;
+        for(var i=0; i < args.length; i++)
+            if (data && (args[i] in data))
+                data = data[args[i]];
+            else
+                return false;
+        return data;
+    }
+    /* end of otb-get-data section */
+
+    /* start of overview-graph config */
+    var graphs = [
+        {plot: 'rx'}, {plot: 'tx'}
+    ];
+
+    interface_colors = (str) => {
+        if(str == 'if1')
```

Comment:
We should try to avoid hardcoded names

Reply:
We must define base interfaces names to keep graph readable and colours consistency in the whole UX, hard-coded link between interface name and colours is hard to avoid.

Reply:
We could use the name to get a color, a hashing function might help solve this problem

Reply:
a very simple solution is to do something like color(name)=colorify(hash(name)).

Reply:
i have taken colours from here :
http://ksrowell.com/blog-visualizing-data/2012/02/02/optimal-colors-for-graphs/

Reply:
Done, but for now i am using a colour bank model instead of a string colorify/hash libs.